permissions/plugin-aqua-microscanner.yml

```diff
@@ -5,3 +5,7 @@ paths:
 - "org/jenkins-ci/plugins/aqua-microscanner"
 developers:
 - "oranmoshai"
+security:
+  contacts:
+    jira: oranmoshai
+    email: maintainers@aquasec.com
```

Comment:
Per this repo's README, this adds an overhead to our processes we're only willing to support for large organizations with dedicated product security teams we would need to coordinate with (e.g. because of CNA scope overlap). This does not seem to be the case for your organization. Please explain why this is necessary, or remove the email contact.

If you want to decouple personal accounts from security contacts, register a Jenkins community account backed by a team email address and specify that account here.

Reply:
I will register team community account thank you

Reply:
@oranmoshai Thanks for the update.

Could you clarify whether you will update this PR with a different account for the security contact, or whether it is final? As it is now, the security contact is explicitly specified but otherwise a "no-op", as I would just contact one of the maintainers if undefined.

Reply:
I updated the final team user for security contact thank you for helping to create good process